llvm/test/CodeGen/AMDGPU/agpr-to-agpr-copy.mir

```diff
@@ -1,5 +1,6 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
 # RUN: llc -mtriple=amdgcn -mcpu=gfx908 -run-pass=liveintervals,amdgpu-pre-ra-optimizations -verify-machineinstrs %s -o - | FileCheck -check-prefix=GFX908 %s
+# RUN: llc -mtriple=amdgcn -mcpu=gfx908 -passes="require<live-intervals>,amdgpu-pre-ra-optimizations" %s -o - | FileCheck -check-prefix=GFX908 %s
```

Comment:
Nit: Should we really need to enforce the require<live-inervals>? The analysis if not available will get computed anyway. This should also be fixed in the legacy run line.